internal/command/status/machines.go

```diff
@@ -31,6 +37,47 @@ func renderMachineStatus(ctx context.Context, app *api.AppCompact) error {
 		return renderPGStatus(ctx, app, machines)
 	}
 
+	// Tracks latest eligible version
+	var latest *api.ImageVersion
+
```

Comment:
Doesn't look like this is working.
`fly image show`
![Screen Shot 2022-10-10 at 12 24 11 PM](https://user-images.githubusercontent.com/423038/194921831-db870783-d33f-48c0-8bdf-98067f933aec.png)

`fly status`
![Screen Shot 2022-10-10 at 12 25 23 PM](https://user-images.githubusercontent.com/423038/194921758-5a4f1ddb-9247-4c4a-a686-68535c86c3e3.png)